libraries/compile-examples/Dockerfile

```diff
@@ -7,6 +7,7 @@ CMD /bin/bash
 
 # Copies your code file from your action repository to the filesystem path `/` of the container
 COPY entrypoint.sh /entrypoint.sh
+COPY reportsizetrends /reportsizetrends
 
 # Code file to execute when the docker container starts up (`entrypoint.sh`)
 ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
I would suggest to install all the dependencies required both for the python part and the bash part directly in the container image, replacing the whole Dockerfile with this version
```
FROM python:3.8.2

# Install prerequisites
RUN apt-get update && apt-get install -y git wget jq curl \
    && rm -rf /var/lib/apt/lists/*

# Copies your code file from your action repository to the filesystem path `/` of the container
COPY entrypoint.sh /entrypoint.sh
COPY reportsizetrends /reportsizetrends

# Install python dependencies
RUN pip install -r /reportsizetrends/requirements.txt

# Code file to execute when the docker container starts up (`entrypoint.sh`)
ENTRYPOINT ["/entrypoint.sh"]
```

Reply:
Thanks @rsora! I have done this, in addition to removing the dependencies installations from entrypoint.sh, in https://github.com/arduino/actions/pull/18/commits/c09f5b4864625f15d01036713f26ba89280bfacc